pom.xml

```diff
@@ -210,6 +211,51 @@
     </pluginManagement>
   </build>
 
+  <profiles>
+    <profile>
+      <id>java9</id>
+      <activation>
+        <jdk>[9,)</jdk>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <version>3.8.0</version>
+            <configuration>
+              <source>8</source>
```

Comment:
just curious if we need this since this profile is for jdk 9+ only?

Reply:
Ah, possibly not. I think I can just use

```
<configuration>
  <release>9</release>
</configuration>
```

Lemme try that.

Reply:
Latest version uses that and seems to be fine with it.

Reply:
I think you can simply remove it? I was looking at https://github.com/googleapis/java-pubsublite/blob/adb22207fb3cba10de95a5e6399456d362408dd7/pom.xml#L166-L199

Reply:
I was under the understanding if you didn't have either `<source>` or `<target>` or `<release>`, the default behavior was that it was treated as Java 6 (i.e., 1.6), but perhaps that was in earlier releases of Java? Anyway, removing it also works.